include/srtp.h

```diff
@@ -69,13 +69,18 @@ extern "C" {
 /*
  * SRTP_MAX_KEY_LEN is the maximum key length supported by libSRTP
  */
-#define SRTP_MAX_KEY_LEN 64
+#define SRTP_MAX_KEY_LEN 96
+
+/*
+ * SRTP_MAX_KEY_LEN is the maximum IV length supported by libSRTP
+ */
+#define SRTP_MAX_IV_LEN 24
```

Comment:
Why is this in public header file? 
Also the comment is wrong.

Reply:
It's in the public header file to be parallel to `SRTP_MAX_KEY_LEN`, `SRTP_MAX_TAG_LEN`, etc.  Do you think that because the IV is generated by libsrtp, it should go in the private header?  Happy to move it, just want to understand the rationale.

How is the comment wrong?

Reply:
SRTP_MAX_KEY_LEN  => SRTP_MAX_IV_LEN  in the comment

Yes that was my rational but I am ok with it staying here.